style.css

```diff
@@ -470,3 +491,293 @@ main {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 12px;
+    position: absolute;
+    width: 1375px;
+    top: 24px;
+    background-image: none;
+    background-image: url(resources/img/Header-bg.png);
+    background-position: center center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 769px;
+    background-origin: content-box;
+  }
+
+  .main-content-desktop {
+    position: absolute;
+    top: 180px;
+    left: 230px;
+  }
+
+  .name {
+    width: 60vw;
+    font-weight: 700;
+    font-size: 3rem;
+    line-height: 50px;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    margin-top: 114px;
+    margin-bottom: 16px;
+  }
+
+  .self-intro {
+    width: 50vw;
+    height: 148px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 26px;
+    color: #344563;
+    margin-bottom: 10px;
+  }
+
+  .social-connect {
+    margin-bottom: 20px;
+  }
+
+  main {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 2694px;
+    gap: 10px;
+  }
+
+  .works {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: center;
+    margin: 50px 24px;
+    padding: 16px;
+    width: 86vw;
+    height: 552px;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+  }
+
+  .snapshoot img {
+    display: none;
+  }
+
+  .snapshoot-desktop {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    padding: 44px;
+    gap: 24px;
+  }
+
+  .workss {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: center;
+    width: 100%;
+    height: 634px;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-top-right-radius: 124px;
+    padding: 64px 0;
+  }
+
+  .workss-wrapper {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: flex-start;
+    gap: 94px;
+    position: absolute;
+    left: 142px;
+    width: 80vw;
+  }
+
+  .name2 {
+    width: 60vw;
+    font-weight: 700;
+    font-size: 3rem;
+    line-height: 50px;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    margin-top: 14px;
+    margin-bottom: 16px;
+  }
+
+  .about-me p {
+    width: 400px;
+  }
+
+  .footer-wrap {
+    background-color: hsl(234, 100%, 69%);
+    border-top-left-radius: 84px;
+    background-image: url(resources/img/contact\ formbackgroundshapes@2x.png);
+    background-position: right 24px;
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 779px;
+    display: flex;
+    flex-direction: column;
+    background-origin: content-box;
+    padding: 75px;
+  }
+
+  .contact-form {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .tech-toolss {
+    display: flex;
+    align-items: flex-start;
+    justify-content: flex-start;
+    background-color: #fff;
+  }
```

Comment:
This section doesn't quite look like the one from the template design.

Here's your design:

<img width="343" alt="Screen Shot 2022-06-02 at 19 23 30" src="https://user-images.githubusercontent.com/82426682/171761243-eecc22ff-f6a7-40b2-a7b2-87551460e7a8.png">

Here's your template design:

<img width="408" alt="Screen Shot 2022-06-02 at 19 24 12" src="https://user-images.githubusercontent.com/82426682/171761300-b72097f3-676f-4ecb-88b1-3e43005f4283.png">

Kindly fix it.